scripts/test_suite.sh

```diff
@@ -12,7 +12,7 @@ reportvalidation() {
 }
 
 
-python -m pytest -vv --durations=3 --cov=spylib --cov-report term-missing
+python -m pytest -rs -vv --durations=3 --cov=spylib --cov-report term-missing
```

Comment:
Why `-rs`?

Reply:
was having skipped tests. not any more I will get rid of it